vmm/Cargo.toml

```diff
@@ -58,6 +58,8 @@ landlock = "0.4.2"
 libc = { workspace = true }
 linux-loader = { workspace = true, features = ["bzimage", "elf", "pe"] }
 log = { workspace = true }
+# Special fork of micro_http that combines HTTP traffic over a UNIX domain
+# socket with UNIX' SCM_RIGHTS mechanism for transferring file descriptors.
```

Comment:
Not sure about the meaning of "special fork", and why we need to emphasize on `SCM_RIGHTS` support here. It is simply of the features that the `micro_http` crate supports on Linux environment, and I don't think that's the main reason why it was picked.

You need to support it for anything that needs to send FDs across processes, say vfio-user and vhost-user crates also support it.

Reply:
fixed